content/rejectedData.ttl

```diff
@@ -0,0 +1,7 @@
+@prefix : <http://glossary.orcfax.io/#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+:rejectedData
+    skos:prefLabel "rejected data"@en;
+    skos:definition '''
+    Data that was received by an Orcfax validator but rejected as it did not have all of the properties required to satisfy the consensus algorithm, e.g. its content signature did not match what was received or the median could not be accurately re-calculated or its timestamp was > 180 seconds.
```

Comment:
```suggestion
    Data that was received by an Orcfax validator but rejected as it did not have all of the properties required to satisfy the consensus algorithm, e.g. its content signature did not match what was received or the median could not be accurately re-calculated or its timestamp was > 80 seconds.
```

ref: https://github.com/orcfax/validator-node-api/blob/35c7a6f24cb17b86f6efc857268246baf48670b0/src/validator_node_api/config.py#L62

(I probably lied when I wrote this)